src/components/VmDisks/utils.test.js

```diff
@@ -0,0 +1,76 @@
+/* eslint-env jest */
+
+import { sortDisksForDisplay } from './utils'
+import { fromJS } from 'immutable'
+
+const samples = [
+  {
+    testTitle: 'bootable, simple name',
+    locale: 'en',
+    test: fromJS([
+      { bootable: false, name: 'Zulu' },
+      { bootable: false, name: 'Yankee' },
+      { bootable: true, name: 'Xray' },
+      { bootable: true, name: 'Alpha' },
+      { bootable: false, name: 'Beta' },
+    ]),
+    expect: fromJS([
+      { bootable: true, name: 'Alpha' },
+      { bootable: true, name: 'Xray' },
+      { bootable: false, name: 'Beta' },
+      { bootable: false, name: 'Yankee' },
+      { bootable: false, name: 'Zulu' },
+    ]),
+  },
+
+  {
+    testTitle: 'name with number',
+    locale: 'en',
+    test: fromJS([
+      { bootable: false, name: 'Zulu' },
+      { bootable: false, name: 'Yankee 7' },
+      { bootable: false, name: 'Yankee' },
+      { bootable: false, name: 'Xray' },
+      { bootable: false, name: 'Alpha 1' },
+      { bootable: false, name: 'Alpha 10' },
+      { bootable: false, name: 'Alpha 7' },
+      { bootable: false, name: 'Beta' },
+    ]),
+    expect: fromJS([
+      { bootable: false, name: 'Alpha 1' },
+      { bootable: false, name: 'Alpha 7' },
+      { bootable: false, name: 'Alpha 10' },
+      { bootable: false, name: 'Beta' },
+      { bootable: false, name: 'Xray' },
+      { bootable: false, name: 'Yankee' },
+      { bootable: false, name: 'Yankee 7' },
+      { bootable: false, name: 'Zulu' },
+    ]),
+  },
+
+  {
+    testTitle: 'name with number',
+    locale: 'cs',
```

Comment:
Note, action not required

---

```js
['r', 'ř10', 'ř9', 'ř'].sort((a, b) => a.localeCompare(b, 'cs', { numeric: true }))
['r', 'ř10', 'ř9', 'ř'].sort((a, b) => a.localeCompare(b, 'en', { numeric: true }))
```

both returns the same result `Array [ "r", "ř", "ř9", "ř10" ]`. So this isn't actually testing that other sorting algorithm is used. The distiguishing difference might be the fact that Czech language consider sequence 'ch' as one char that is sorted between 'h' and 'i':

```js
['cecil', 'chris', 'david', 'harry'].sort((a, b) => a.localeCompare(b, 'cs', { numeric: true }))
// Array [ "cecil", "david", "harry", "chris" ]
['cecil', 'chris', 'david', 'harry'].sort((a, b) => a.localeCompare(b, 'en', { numeric: true }))
// Array [ "cecil", "chris", "david", "harry" ]
```

Reply:
Thanks for the language lesson!  I'll adjust the tests.

Reply:
Oh fun.  The normal build of nodejs, at least for node 8 and newer, appears to enable the Intl API but only includes English (en).  There is a way to include the full locale support.  I'll update package.json to do that, however that will also require updating `ovirt-engine-nodejs-modules` for full CI and build support.